engine/gui/src/gui_private.h

```diff
@@ -105,72 +105,65 @@ namespace dmGui
 
     struct Node
     {
-        dmVMath::Vector4    m_Properties[PROPERTY_COUNT];
-        dmVMath::Vector4    m_ResetPointProperties[PROPERTY_COUNT];
-        dmVMath::Matrix4    m_LocalTransform;
-        dmVMath::Vector4    m_LocalAdjustScale;
-        uint32_t            m_ResetPointState;
+        dmVMath::Matrix4        m_LocalTransform;
+        dmVMath::Vector4        m_Properties[PROPERTY_COUNT];
+        dmVMath::Vector4        m_LocalAdjustScale;
 
-        uint32_t    m_HasResetPoint         : 1; // If true, we have stored a copy of the m_State, into m_ResetPointState
-        uint32_t    m_PerimeterVertices     :31;
-        PieBounds   m_OuterBounds;
+        uint64_t                m_TextureHash;
+        uint64_t                m_FlipbookAnimHash;
+        uint64_t                m_FontHash;
+        uint64_t                m_ParticlefxHash;
+        dmhash_t                m_LayerHash;
+        dmhash_t                m_MaterialNameHash;
```

Comment:
I wonder if we should change all the hash to use `dmhash_t` here, or if it causes problems?
I don't want the PR to grow because of that. 🤔

Reply:
if you don't mind, let's keep it for now as is, and fix it in the future iterations